packages/neuron-wallet/src/services/settings.ts

```diff
@@ -88,7 +88,7 @@ export default class SettingsService extends Store {
 
   private generateEncryptString(str: string) {
     if (safeStorage.isEncryptionAvailable()) {
-      return safeStorage.encryptString(str).toString('utf-8')
+      return safeStorage.encryptString(str).toString('hex')
```

Comment:
this change might cause the decryption to fail because the decrypted string is no longer the same as before

Reply:
This function has not been released, it's a new future in the next version. So I guess it does not matter.

Reply:
Got it. Can we return the raw buffer instead of the encoded string? Then compare the password with `Buffer.prototype.equals`

Reply:
The encoded string needs to be cached on the disk. So it needs to transfer to a string.